prow/jobs/incubator/compass/tests/provisioner-tests/provisioner-whitesource.yaml

```diff
@@ -0,0 +1,54 @@
+test_infra_ref: &test_infra_ref
+  org: kyma-project
+  repo: test-infra
+  path_alias: github.com/kyma-project/test-infra
+
+
+whitesource_job_template: &whitesource_job_template
+  skip_report: false
+  run_if_changed: "go.mod|go.sum"
+  decorate: true
+  optional: true
+  path_alias: github.com/kyma-incubator/compass/components/provisioner
+  max_concurrency: 2
+  labels:
+    preset-dind-enabled: "true"
+    preset-docker-push-repository-kyma: "true"
+    preset-build-pr: "true"
+    preset-kyma-wssagent-config: "true"
+    preset-kyma-keyring: "true"
+    preset-kyma-encryption-key: "true"
+    preset-kms-gc-project-env: "true"
+    preset-gc-project-env: "true"
+    preset-sa-gke-kyma-integration-kyma-project-whitesource: "true"
```

Comment:
I don't know if you need `docker-push` or `dind`, so you could probably remove these labels